spec/factories/calendar_events_factory.rb

```diff
@@ -17,7 +17,7 @@
       description { 'this is a new term' }
       association :calendar_event_type, term_time: true, holiday: false, title: 'Term 1'
     end
-    factory :holiday do
+    factory :calendar_event_holiday do
```

Comment:
perhaps should rename the analytics factory instead :thinking:

Reply:
I think this is fine